src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -391,18 +409,26 @@ impl<'b> Resolver<'b> {
         }
 
         let name = xcdef.name;
-        let vis = if parent.is_trait() { ty::Visibility::Public } else { xcdef.vis };
+        let vis = if parent.is_trait() {
+            ty::Visibility::Public
+        } else {
+            xcdef.vis
+        };
```

Comment:
"Conditional operator" in Rust is already noisy as hell compared to C's `?:` and rustfmt makes it even worse. (5)